recipes-core/searcher-ssh-key/files/searcher-ssh-key.sh.in

```diff
@@ -9,8 +9,6 @@
 ### END INIT INFO
 
 start() {
-  # Start the attested TLS reverse proxy server
-  start_proxy
```

Comment:
Is it no longer needed?

Reply:
No, it is a left over from pervious commits. 
It is moved to the ssh http server init file